### PR TITLE
simplify options page for now, save API key

### PR DESCRIPTION
**Pull request by kevintpeng**
use `chrome.storage` to store and reference our API key @EssaSaeed @n7huang

### CODE REVIEW

File: src/options/index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+<head><title>My Test Extension Options</title></head>
```
Comment (EssaSaeed) [inline src/options/index.html]: Might as well call it "Summarizer Options"

File: src/options/options.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    setTimeout(function() {
+      status.textContent = '';
+    }, 750);
+  });
+}
+
+// Restores select box and checkbox state using the preferences
+// stored in chrome.storage.
+function restore_options() {
+  // Use default value color = 'red' and likesColor = true.
```
Comment (EssaSaeed) [inline src/options/options.js]: I don't think this comment's supposed to be here

**Comment by kevintpeng** [comment]
@EssaSaeed how does this look?

**Comment by EssaSaeed** [comment]
@kevintpeng looking good 👍

**Comment by kevintpeng** [comment]
Thanks for the review Essa!

**Pull request merged**